pkg/plugin/plugin.go

```diff
@@ -18,9 +18,65 @@ const (
 	MethodSendNotification = "SendNotification"
 )
 
+// ConfigOption describes a config element of the channel form
+type ConfigOption struct {
+	// Element name
+	Name string `json:"name"`
+
+	// Element type:
+	//
+	//  string = text, port = number, bool = checkbox, text = textarea, option = select, options = select[multiple], secret = password
+	Type string `json:"type"`
```

Comment:
Please explain why you changed "int" to "port" as that looks quite inconsistent compared to the others.

Reply:
Johannes suggested me to do it this way. This creates an input element on the `notifications-web` that only accepts `0 - 65535`.
He said that these options should not define the form elements exactly. They are just options from which we can (somehow) generate form elements. The other idea would be to use the type `number = number` instead and two extra optional options `min` and `max`.

Reply:
"number" with min/max sounds more flexible to me. If there was a type "port", it should probably at most be an addition to a generic number/integer field, not replace it.

Sure, there can be reasons for more specific types, the question is always where it starts becoming too much complexity for the benefits. Sure, we could define types like hostname/address, url, e-mail, ..., but do we want to specify and maintain that list?